### DISCUSSION TITLE
Elevation problem

### DISCUSSION THREAD

**Discussion by marcoma9023** [pelicanmapping/osgearth#1871] [Support Q&A]
Hi,

launching the osgearth_elevation.exe I can't see the terrain elevation, why?

**Comment by keince** [comment]
Try running the application using an earth file...

osgearth_elevation readymap.earth

You should be able to see elevation with imagery draped over the terrain if you zoom in and then rotate by holding the middle mouse button while moving the mouse.

HTH...

**Reply by marcoma9023** [reply]
Thanks for reply,

but unfortunately I don't have elevation again....

![image](https://user-images.githubusercontent.com/87537804/142448907-89c16bf8-5de5-4dc9-9ea4-b35930a00016.png)

**Reply by keince** [reply]
Strange that you can load ReadyMap imagery but not elevation. Can you provide the output from your console window to show any errors being reported? Before you do this, it would be helpful for you to set the environment variable OSGEARTH_NOTIFY_LEVEL=INFO

**Reply by nmielcarek** [reply]
Which version of osgEarth are you running?

In the version I'm using (2.10) I had to add the following line to src/osgEarth/Registry.cpp:
CPLSetConfigOption("OGR_CT_FORCE_TRADITIONAL_GIS_ORDER", "YES");

I believe this was due to a change in the latest GDAL library.

**Comment by marcoma9023** [comment]
> Strange that you can load ReadyMap imagery but not elevation. Can you provide the output from your console window to show any errors being reported? Before you do this, it would be helpful for you to set the environment variable OSGEARTH_NOTIFY_LEVEL=INFO

void StateSet::setGlobalDefaults() ShaderPipeline disabled.
[osgEarth]  Loading href from C:/Marco/USA/Visual Studio/osgearth-master/build2/bin/Release/viewpoints.xml
[osgEarth]  [EarthSerializer2] Loaded extension lib "osgPlugins-3.7.0/osgdb_osgearth_viewpoints.dll"
[osgEarth]  [Map] cache=none; policy=read-write; bin=no
[osgEarth]  [TerrainLayer] Layer "readymap_elevation" Creating "tms" driver
[osgEarth]  [TMSTileSource] TMS tile map datestamp = Thu, 01 Jan 1970 00:00:00 GMT
[osgEarth]  override vdatum = egm96, profile vdatum =
[osgEarth]  [VerticalDatum] Initializing vertical datum: egm96
[osgEarth]  [TerrainLayer] Layer "readymap_elevation" Override profile: [srs=WGS 84, min=-180,-90 max=180,90 lod0=2,1 vdatum=EGM96]
[osgEarth]  [TerrainLayer] Layer "readymap_elevation" Profile=[srs=WGS 84, min=-180,-90 max=180,90 lod0=2,1 vdatum=EGM96]
[osgEarth]  [TerrainLayer] Layer "readymap_elevation" cache=none; policy=read-write; bin=no
[osgEarth]  [TerrainLayer] Layer "readymap_imagery" Creating "tms" driver
[osgEarth]  [TMSTileSource] TMS tile map datestamp = Thu, 01 Jan 1970 00:00:00 GMT
[osgEarth]  [TerrainLayer] Layer "readymap_imagery" Profile=[srs=WGS 84, min=-180,-90 max=180,90 lod0=2,1 vdatum=geodetic]
[osgEarth]  [TerrainLayer] Layer "readymap_imagery" cache=none; policy=read-write; bin=no
[osgEarth]  [ViewpointsExtension] Read 8 viewpoints
[osgEarth]  [engine_rex] Activated!
[osgEarth]  [Capabilities] osgEarth Version: 2.10.2
[osgEarth]  [Capabilities] OSG Version:      3.7.0
[osgEarth]  [Capabilities] GDAL Version:     2.1.3
[osgEarth]  [Capabilities] Detected hardware capabilities:
[osgEarth]  [Capabilities]   Vendor = NVIDIA Corporation
[osgEarth]  [Capabilities]   Renderer = GeForce GTX 1060 3GB/PCIe/SSE2
[osgEarth]  [Capabilities]   Version = 4.6.0 NVIDIA 457.51
[osgEarth]  [Capabilities]   Core Profile = no
[osgEarth]  [Capabilities]   Max GPU texture units = 32
[osgEarth]  [Capabilities]   Max GPU texture coord indices = 8
[osgEarth]  [Capabilities]   Max GPU attributes = 16
[osgEarth]  [Capabilities]   Depth buffer bits = 24
[osgEarth]  [Capabilities]   Max texture size = 32768
[osgEarth]  [Capabilities]   GLSL = yes
[osgEarth]  [Capabilities]   GLSL Version = 460
[osgEarth]  [Capabilities]   Texture arrays = yes
[osgEarth]  [Capabilities]   draw instanced = yes
[osgEarth]  [Capabilities]   Texture buffers = yes
[osgEarth]  [Capabilities]   Texture buffer max size = 134217728
[osgEarth]  [Capabilities]   Compression = ARB S3 RG
[osgEarth]  [Map] Map profile is: [srs=WGS 84, min=-180,-90 max=180,90 lod0=2,1 vdatum=geodetic]
[osgEarth]  [PagerLoader] Merges per frame = 20
[osgEarth]  [TerrainResources] Texture unit 0 reserved for Terrain Color
[osgEarth]  [TerrainResources] Texture unit 1 reserved for Terrain Elevation
[osgEarth]  [TerrainResources] Texture unit 2 reserved for Terrain Normals
[osgEarth]  [TerrainResources] Texture unit 3 reserved for Terrain Parent Color
void StateSet::setGlobalDefaults() ShaderPipeline disabled.

**Comment by gwaldron** [comment]
Make sure that you have the libtiff plugin built for OpenSceneGraph.
(Someday you should upgrade GDAL -- version 2.1 is old.)
